templates/exporter/exporter-dpl.yaml

```diff
@@ -82,6 +82,13 @@ spec:
               name: {{ .Values.database.external.existingSecret }}
               key: password
         {{- end }}
+        {{- if .Values.database.internal.existingSecret }}
```

Comment:
may need to change `{{- if .Values.database.internal.existingSecret }}` to `{{- if and (eq .Values.database.type "internal") .Values.database.internal.existingSecret -}}`

Reply:
also may also update line 78, `{{- if .Values.database.external.existingSecret }}` -> `{{- if and (eq .Values.database.type "external") .Values.database.external.existingSecret }}`